kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -363,6 +363,8 @@
       const store = currentInstance.$store;
       const router = currentInstance.$router;
       const topic = ref(null);
+      const { fetchContentNodeProgress, fetchContentNodeTreeProgress } = useContentNodeProgress();
```

Comment:
Any reason to move these? Seems like we could have zero diff here.

Reply:
Whoops We had moved this because previously we understood that we needed to usue provide/inject to use these values :sweat_smile:, and I didnt reverted this.